src/components/section/NewEventCard.tsx

```diff
@@ -0,0 +1,198 @@
+import type { ListboxOption } from "../ui/Input";
+import { RoundedListbox, Selector } from "../ui/Input";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import { getCurrentTimeZoneTag, makeTime, moveTime } from "../../utils/utils";
+import { RoundedTimezoneInput } from "../ui/TimezoneInput";
+import { DateSelect } from "../ui/DateSelect";
+import { TimespanSelector } from "../ui/TimeSelector";
+import { ButtonWithState } from "../ui/Button";
+import { useCreateEvent, useEventWithJoinCode } from "../../utils/api-hooks";
+
+const selectDaysOptions: ListboxOption[] = [
+  {
+    label: "Days of week",
+    value: "DAYSOFWEEK",
+  },
+  {
+    label: "Specific days",
+    value: "DATES",
+  },
+];
+
+const StartNewEventSection = () => {
+  const router = useRouter();
+
+  // >>> Form values >>>
+  const [eventName, setEventName] = React.useState("");
+  const [timeZoneTag, setTimeZoneTag] = React.useState(getCurrentTimeZoneTag());
+  const [selectDaysType, setSelectDaysType] = React.useState<
+    "DAYSOFWEEK" | "DATES"
+  >("DAYSOFWEEK");
+  const [selectedDays, setSelectedDays] = React.useState<Date[]>([]);
+  const [startTime, setStartTime] = React.useState<Date>(
+    makeTime(0, 8, 0, timeZoneTag)
+  );
+  const [endTime, setEndTime] = React.useState<Date>(
+    makeTime(0, 22, 0, timeZoneTag)
+  );
+
+  console.log(startTime, endTime);
+
+  // <<< Form values <<<
+
+  const canSubmit = eventName.length > 0 && selectedDays.length > 0;
+
+  const { mutateAsync: createEvent, isLoading: creatingEvent } =
+    useCreateEvent();
+  const handleCreateEvent = async () => {
+    const { id: eventId } = await createEvent({
+      occuringDays: selectedDays.toString(),
+      name: eventName,
+      begins: startTime.toISOString(),
+      ends: endTime.toISOString(),
+      type: selectDaysType,
+      timeZone: timeZoneTag,
+    });
+    await router.push(`/event/${eventId}`);
+  };
+
+  return (
+    <>
+      <div className="input-field text-sm">
+        <label>Event name</label>
+        <input
+          className="rounded-input"
+          placeholder="New Meeting"
+          onChange={(e) => setEventName(e.target.value)}
+        />
```

Comment:
Should we do a "non-empty" check here? cuz now I'm able to enter a " " space as event name and then the list will look like dis:
<img width="545" alt="Screenshot 2023-06-18 at 13 49 49" src="https://github.com/ParachuteTeam/Parachute/assets/20884681/c9861aed-f2f7-4d41-82b6-e198b193e3b3">

I don't feel like this is intended is it?

Reply:
I fixed this by trimming the event name.